lib/bundler/installer.rb

```diff
@@ -75,6 +75,10 @@ def run(options)
         return
       end
 
+
+      # EXPERIMENT: Evaluate benefit if we skip resolution by comparing Gemfile checksum against the checksum saved in the lockfile.
+      return if Definition.build(Bundler.default_gemfile, Bundler.default_lockfile, nil).checksum_matches?
```

Comment:
I already mentioned to @segiddins but for documentation purposes, this did not result in any time improvement.